spec/workers/every_morning_worker_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'fugit'
+
+RSpec.describe Schedule::EveryMorningWorker, type: :worker do
```

Comment:
you might test its functionality too
checkout this https://relishapp.com/rspec-staging/rspec-rails/docs/matchers/have-enqueued-mail-matcher